outputs.tf

```diff
@@ -1,26 +1,16 @@
-output "emr_managed_master_sg_id" {
-  value       = module.emr-sgs.emr_managed_master_sg_id
-  description = "Security group id of the EMR Managed Master Security Group"
+output "emr_managed_master_sg_ids" {
+  value       = var.emr_managed_master_sg_ids
```

Comment:
We should be returning module.emr-sgs.emr_managed_sg_id somehow in this output. Maybe it's own dedicated variable, but maybe it belongs here in this list and the core list.

Reply:
done